app/models/hostgroup_ansible_role.rb

```diff
@@ -1,5 +1,6 @@
 # Join model that hosts the connection between hostgroups and ansible_roles
 class HostgroupAnsibleRole < ApplicationRecord
+  audited :associated_with => :hostgroup, :allow_mass_assignment => true
```

Comment:
Apparently this needs some change to log the name of the role too

Reply:
the problem is in Foreman core or better to say inability to extend `AuditsHelper::MAIN_OBJECTS` from plugin, Kavita is working on it right now. We can merge as it is now so we start getting data and later we just change the way how it's presented. Or we can wait few days until the core DSL is available, though I'd prefer the first approach.